its/core-it-support/maven-it-helper/src/main/java/org/apache/maven/it/Verifier.java

```diff
@@ -594,22 +594,22 @@ public void verifyTextNotInLog(String text) throws VerificationException, IOExce
     }
 
     public static void verifyTextNotInLog(List<String> lines, String text) throws VerificationException {
-        if (textOccurencesInLog(lines, text) > 0) {
+        if (textOccurrencesInLog(lines, text) > 0) {
             throw new VerificationException("Text found in log: " + text);
         }
     }
 
     public static void verifyTextInLog(List<String> lines, String text) throws VerificationException {
-        if (textOccurencesInLog(lines, text) <= 0) {
+        if (textOccurrencesInLog(lines, text) <= 0) {
             throw new VerificationException("Text not found in log: " + text);
         }
     }
 
     public long textOccurrencesInLog(String text) throws IOException {
-        return textOccurencesInLog(loadLogLines(), text);
+        return textOccurrencesInLog(loadLogLines(), text);
     }
 
-    public static long textOccurencesInLog(List<String> lines, String text) {
+    public static long textOccurrencesInLog(List<String> lines, String text) {
```

Comment:
This is a public method. Not sure we can just rename it. Might have to deprecate and replace depending on when it was added